elastic/kibana
# [Stack Monitoring] Document count graph doesn't show exact number

closes https://github.com/elastic/kibana/issues/164667

## 📝  Summary

This PR modifies the legend for `Document Count` graph in stack monitoring to allow showing the exact document count when the user hovers on a specific point.

## ✅  Testing

1. Make sure to have any dummy index with millions of documents
2. Navigate to Indices tab in Stack Monitoring
3. Click on the created index and observe the Document Count Graph 
4. Hovering on any point should show the exact document count through the legend below the graph

## 🎥 Demo

https://github.com/elastic/kibana/assets/11225826/56747cf7-7914-4742-884a-9f9a9f59e9e6
